final-project/README.md

```diff
@@ -5,6 +5,45 @@ This application allows users to sign up for an account, sign in, record and man
 
 The server and client are both written on Go, and run on Docker.
 
+## Project Directory
+```
+final-project/
+│   Taskfile.yml
+│   go.mod
+│   main.go                     - main file for running the server
+│   README.md
+│   routes.go
+│
+└───client/                     - module for client app
+│   
+└───deploy/
+│   │
+│   └───dev/
+│       │   
+│       └───data/
```

Comment:
You may have to have some files for redis and some files for the "file" storage.
But it's not important.